.github/workflows/build-then-deploy-docker.yml

```
@@ -0,0 +1,62 @@
+# Copyright 2023 Democratized Data Foundation
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+# This workflow builds a Docker container image, if the build is successful
+# then it will deploy the image to DockerHub & GitHub container registries.
+name: Build Then Deploy Docker Workflow
+
+on: 
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  push-to-registries:
+    name: Push Docker image to multiple registries
```

todo: We usually put `job` in the ending of every job's name (helps distinguish steps, workflows, and jobs in various places).

```suggestion
    name: Push docker image to registries job
```